src/components/UTIL_PageMessages.component

```diff
@@ -38,8 +38,8 @@
                             <svg class="slds-button__icon slds-button__icon_large" aria-hidden="true">
                                 <path d="M14.6 11.9l6-6c.3-.3.3-.7 0-1l-.9-1c-.3-.3-.7-.3-1 0L12.6 10c-.1.2-.4.2-.6 0L6 3.9c-.3-.3-.7-.3-1 0l-1 .9c-.3.3-.3.7 0 1l6.1 6.1c.1.1.1.4 0 .6L4 18.6c-.3.3-.3.7 0 1l.9.9c.3.3.7.3 1 0l6.1-6c.2-.2.5-.2.6 0l6.1 6c.3.3.7.3 1 0l.9-.9c.3-.3.3-.7 0-1l-6-6c-.2-.2-.2-.5 0-.7z"/>
                             </svg>
-                            <span class="slds-assistive-text">Close</span>
-                        </button> 
+                            <span class="slds-assistive-text">{!m.severityLabel}: {!m.summary} {!$Label.bdiBtnClose}</span>
```

Comment:
Can we use a getter method to return assistive text? Also, the ":" is not recommended to be used outside of the custom labels for the translation requirements.

Reply:
@RadSFDeveloper Requested changes have been made.  Thanks!!